programs/mango-v4/tests/cases/test_health_compute.rs

```diff
@@ -68,7 +68,7 @@ async fn test_health_compute_tokens() -> Result<(), TransportError> {
     let avg_cu_increase = cu_measurements.windows(2).map(|p| p[1] - p[0]).sum::<u64>()
         / (cu_measurements.len() - 1) as u64;
     println!("average cu increase: {avg_cu_increase}");
-    assert!(avg_cu_increase < 3200);
+    assert!(avg_cu_increase < 3230);
```

Comment:
should measure the CU needed when a fallback oracle is needed (the logging might be killer)

Reply:
Logging is indeed really expensive. I'm seeing a cost of ~16000 CU for each fallback used, with an additional ~11000 if the fallback is also bad

Reply:
That sounds almost unbelievably high!